package.json

```diff
@@ -67,6 +67,9 @@
 		"vscode-test": "^1.3.0"
 	},
 	"dependencies": {
+		"bzip2": "^0.1.1",
+		"decompress": "^4.2.1",
```

Comment:
remove this library if it's not being used

Reply:
Your wish is my command, it shall be done.